### PR TITLE
PT-12187 - Add feature: Update pseudo price

### PR DESCRIPTION
**Pull request by DennisGarding**

### CODE REVIEW

File: Resources/snippets/backend/swag_tax/main.ini [hunk 1]
```diff
@@ -16,6 +16,8 @@ wizard/information_panel/settings = "Settings"
 first_card/recalculatePrices = "Keep gross prices"
 first_card/recalculatePrices/support = "In order to maintain gross prices, the net prices are changed. If you use net prices in the product listing (for example with B2B customer groups), these will also change."
 first_card/text = "<a target=\"blank\" href=\"https://docs.shopware.com/en/shopware-5-en/plugins/Change-taxes\">Link to the documentation</a><br /><br />You can use this function to adjust existing tax rates. To make this as easy as possible, a wizard will guide you through several steps. Finally, you can determine whether the changes should be made immediately or at a scheduled time.<br><br>First select whether you want to keep the gross prices and thus change the net prices.<br><br>If the check mark is not set, the net prices will not be changed. <br /><br /><b>Note:</b> If customer groups inherit prices from other customer groups, this inheritance remains. If you change prices of the source customer group, the prices of the inheriting customer group will also change!"
+first_card/recalculatePseudoPrices = "Pseudo prices also adapt"
```
Comment (mitelg) [inline Resources/snippets/backend/swag_tax/main.ini]:
```suggestion
first_card/recalculatePseudoPrices = "Also adjust pseudo price"
```

File: Resources/snippets/backend/swag_tax/main.ini [hunk 2]
```diff
@@ -16,6 +16,8 @@ wizard/information_panel/settings = "Settings"
...
+first_card/recalculatePseudoPrices/support = ""
```
Comment (mitelg) [inline Resources/snippets/backend/swag_tax/main.ini]: no support text in english?

File: Resources/snippets/backend/swag_tax/main.ini [hunk 3]
```diff
@@ -67,6 +69,8 @@ wizard/information_panel/settings = "Einstellungen"
...
 first_card/text = "<a target=\"blank\" href=\"https://docs.shopware.com/de/shopware-5-de/plugins/Steuern-aendern\">Link zur Dokumentation</a><br /><br />Über diese Funktion können Sie die Anpassung von bestehenden Steuersätzen vornehmen. Für eine möglichst einfache Bedienung erfolgt dies über mehrere Schritte innerhalb eines geführten Assistenten. Abschließend können Sie festlegen, ob die Änderungen sofort oder zu einem geplanten Zeitpunkt durchgeführt werden sollen.<br><br>Wählen Sie zunächst aus, ob Sie die Bruttopreise beibehalten möchten und somit die Nettopreise geändert werden. Wenn der Haken nicht gesetzt wird, bleiben die Nettopreise bestehen. <br /><br /><b>Beachten Sie:</b> Wenn Kundengruppen Preise aus anderen Kundengruppen vererben, so bleibt diese Vererbung bestehen. Ändern Sie Preise der Quell-Kundengruppe, werden sich auch die Preise der vererbenden Kundengruppe ändern!"
+first_card/recalculatePseudoPrices = "Pseudopreise mit anpassen"
+first_card/recalculatePseudoPrices/support = "Die Pseudopreise werden ebenfalls auf der Grundlage des gegebenen Steuersatzes neu Berechnet."
```
Comment (mitelg) [inline Resources/snippets/backend/swag_tax/main.ini]:
```suggestion
first_card/recalculatePseudoPrices/support = "Die Pseudopreise werden ebenfalls, auf der Grundlage des gegebenen Steuersatzes, neu berechnet."
```

**Review by mitelg — CHANGES_REQUESTED** [review]

File: Resources/snippets/backend/swag_tax/main.ini (hunk 2 above)
Comment (DennisGarding) [reply Resources/snippets/backend/swag_tax/main.ini]: ooops.. Done

File: Resources/snippets/backend/swag_tax/main.ini (hunk 1 above)
Comment (DennisGarding) [reply Resources/snippets/backend/swag_tax/main.ini]: Done

File: Resources/snippets/backend/swag_tax/main.ini (hunk 3 above)
Comment (DennisGarding) [reply Resources/snippets/backend/swag_tax/main.ini]: Done

File: plugin.xml [hunk 4]
```diff
@@ -17,8 +17,14 @@
     <compatibility minVersion="5.2.0"/>
 
     <changelog version="REPLACE_GLOBAL_WITH_NEXT_VERSION">
-        <changes lang="en">PT-12186 - Existing tax rates are now selectable.</changes>
-        <changes lang="de">PT-12186 - Bestehende Steuersätze sind nun auswählbar.</changes>
+        <changes lang="en">
+            PT-12186 - Existing tax rates are now selectable.
```
Comment (mitelg) [inline plugin.xml]:
```suggestion
            PT-12186 - Existing tax rates are now selectable;
```

File: plugin.xml [hunk 5]
```diff
@@ -17,8 +17,14 @@
...
+            PT-12187 - Pseudo prices can now also be adjusted.
```
Comment (mitelg) [inline plugin.xml]:
```suggestion
            PT-12187 - Pseudo prices can now also be adjusted;
```

File: plugin.xml [hunk 6]
```diff
@@ -17,8 +17,14 @@
...
+        </changes>
+        <changes lang="de">
+            PT-12186 - Bestehende Steuersätze sind nun auswählbar.
```
Comment (mitelg) [inline plugin.xml]:
```suggestion
            PT-12186 - Bestehende Steuersätze sind nun auswählbar;
```

File: plugin.xml [hunk 7]
```diff
@@ -17,8 +17,14 @@
...
+            PT-12187 - Pseudopreise können nun auch angepasst werden.
```
Comment (mitelg) [inline plugin.xml]:
```suggestion
            PT-12187 - Pseudopreise können nun auch angepasst werden;
```

**Review by mitelg — CHANGES_REQUESTED** [review]

File: plugin.xml (hunk 4 above)
Comment (DennisGarding) [reply plugin.xml]: Done

File: plugin.xml (hunk 5 above)
Comment (DennisGarding) [reply plugin.xml]: Done

File: plugin.xml (hunk 6 above)
Comment (DennisGarding) [reply plugin.xml]: Done

File: plugin.xml (hunk 7 above)
Comment (DennisGarding) [reply plugin.xml]: Done

**Review by mitelg — APPROVED** [review]

**Pull request merged**